core/src/edu/cornell/gdiac/main/PlayerModel.java

```diff
@@ -196,6 +207,16 @@ public boolean isFrozen() {
    */
   public void setFrozen(boolean value) {
     // TODO P3 update frozen and apply physics, also visually change player in some way
+    isFrozen = value;
+    if (isFrozen){
+      setDensity(getDensity() * 3);
+      setFixedRotation(true);
+      color = Color.BLUE;
+    } else {
+      setDensity(getDensity()/3);
```

Comment:
This code breaks the game. Every frame `setFrozen` is being called, so when the player is not frozen the density gets exponentially smaller. This means the player has density 0 after all of these calls and then the movement of the game breaks. Instead, set density to some constant default value. You could make some private final float for `DEFAULT_DENSITY` and then reset it to that.